src/views/SimpleAccount/DepositLists.tsx

```diff
@@ -88,28 +90,19 @@ const V4DepositsList: React.FC<{ usersAddress: string }> = (props) => {
   )
 }
 
-const V3DepositsList: React.FC<{ usersAddress: string }> = (props) => {
-  const { usersAddress } = props
-  const { t } = useTranslation()
-  const { data, isFetched } = useUsersV3PrizePoolBalances(usersAddress)
-
-  if (!isFetched) {
-    return <LoadingList />
-  }
+const V3DepositsList: React.FC<{ usersAddress: string, balances: V3PrizePoolBalances[] }> = (props) => {
+  const { usersAddress, balances } = props
 
   return (
     <ul className='space-y-2'>
-      {data.balances.map((balances) => (
+      {balances.map(balance => (
```

Comment:
It was named `balances` since it returns the token and ticket balances. The outer `balances` should probably be `balancesList` to be clearer.

Reply:
Fixed through d8b2b5b